src/components/MessagePane/index.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react"
+
+const MessagePane = ({
+  id,
+  formatDate,
+  closeButton,
+  setCurrentEmailId,
+  setShowCloseButton,
+}) => {
+  const [currentEmailData, setCurrentEmailData] = useState({})
+
+  const closeEmail = () => {
+    setCurrentEmailId("")
+    setShowCloseButton(false)
+  }
+
+  useEffect(() => {
+    fetch(`https://email-client-api.dev.io-academy.uk/emails/${id}`)
+      .then((response) => response.json())
+      .then((data) => {
+        setCurrentEmailData(data.data.email)
+      })
+  }, [id])
+
+  return (
+    <>
+      {currentEmailData && (
+        <div
+          className={`w-full p-8 ${closeButton}
+        bg-white      
+        sm:w-8/12 
+              w-screen
+              h-screen
+              sm:static 
+              absolute 
+              z-10 
```

Comment:
- Your indentation is all weird.
- `w-full p-8 ${closeButton}` one line, then the rest are on their own lines? Pick one or the other.
- 🤔 Just interesting how you did this all in Tailwind.